app/src/marketplace/modules/deprecated/purchaseDialog/actions.js

```diff
@@ -53,34 +53,7 @@ const setAccessPeriodData: AccessPeriodActionCreator = createAction(
     }),
 )
 
-const getBalances = (): Promise<[BN, BN]> => {
-    const web3 = getWeb3()
-    const ethPromise = getEthBalance(web3)
-    const dataPromise = getDataTokenBalance(web3)
-
-    return Promise.all([ethPromise, dataPromise])
-        .then((results) => {
-            const ethBalance = BN(results[0])
-            const dataBalance = BN(results[1])
-            return [ethBalance, dataBalance]
-        })
-}
-
-export const validateDataBalanceForPurchase = async (price: BN) => {
-    const [ethBalance, dataBalance] = await getBalances()
-
-    const requiredEth = fromAtto(gasLimits.BUY_PRODUCT)
-
-    if (ethBalance.isLessThan(requiredEth) || dataBalance.isLessThan(price)) {
-        throw new NoBalanceError(
-            I18n.t('error.noBalance'),
-            requiredEth,
-            ethBalance,
-            price,
-            dataBalance,
-        )
-    }
-}
+const validateDataBalanceForPurchase = (p) => new Promise(() => true)
```

Comment:
This is still used in the old price dialog, I think? If not, can it be removed?

Reply:
It's still being used, I will revert `getBalances` & `validateDataBalanceForPurchase`.